docs/deployment/heroku.md

```diff
@@ -168,12 +168,13 @@ defmodule HelloWeb.UserSocket do
   ## Transports
   transport :websocket, Phoenix.Transports.WebSocket,
     timeout: 45_000
+    ...
 end
 ```
 
-This ensures that any idle connections are closed by Phoenix before they reach Heroku's 55 second timeout window.
+This ensures that any idle connections are closed by Phoenix before they reach Heroku's 55-second timeout window.
 
-Lastly, we'll need to create a [Procfile](https://devcenter.heroku.com/articles/procfile) with the following:
+Lastly, we'll need to create a [Procfile](https://devcenter.heroku.com/articles/procfile) (a text file called “Procfile” in the root of our project’s folder) with the following line:
```

Comment:
This is pedantic, but could we use `"` instead of `“` to reflect usage in the rest of the guide?

There are 2 other occurrences of this in the guides that I will remove.